src/parser/tests.rs

```diff
@@ -406,17 +396,88 @@ fn label_no_colon_in_context() {
         ))
     );
 }
+
 #[test]
 fn label_in_opcode() {
-    let parsed = super::parse_lines("BEQ LABEL".as_bytes());
+    let parsed = super::parse_lines("BEQ LABEL");
     assert_eq!(
         parsed,
         Ok((
-            &[] as &[u8],
+            "",
             vec![Token::OpCode(OpCode(
                 Mnemonic::Beq,
                 AddressingMode::Label("LABEL".to_string())
             )),]
         ))
     );
 }
+
+#[test]
+fn parse_direct_bytes_only() {
+    let parsed = super::parse_lines(".BYTE $4E, $45, $53, $1A");
+    assert_eq!(
+        parsed,
+        Ok((
+            "",
+            vec![Token::ControlCommand(ControlCommand::Byte(vec![
+                ('N' as u8, Sign::Implied),
+                ('E' as u8, Sign::Implied),
+                ('S' as u8, Sign::Implied),
+                (0x1A, Sign::Implied)
+            ]))]
+        ))
+    );
+}
+
+#[test]
+fn parse_comments() {
+    let parsed_with_comments = super::parse_lines(
+        r#"
+        NOP ; comment
+        NOP; comment
+
+        ; comment
+        NOP
+; comment
+        NOP
+;comment
+    "#,
+    );
+    let parsed_without_comments = super::parse_lines("NOP\nNOP\nNOP\nNOP");
+    assert_eq!(parsed_with_comments, parsed_without_comments);
+}
+
+#[test]
+fn parse_direct_bytes_interleved() {
+    let parsed = super::parse_lines(
+        r#"
+        NOP
+        .BYTE $4E, $45, $53, $1A
+        NOP
+    "#,
+    );
+    assert_eq!(
+        parsed,
+        Ok((
+            "",
+            vec![
+                Token::OpCode(OpCode(Mnemonic::Nop, AddressingMode::Implied)),
+                Token::ControlCommand(ControlCommand::Byte(vec![
+                    ('N' as u8, Sign::Implied),
+                    ('E' as u8, Sign::Implied),
+                    ('S' as u8, Sign::Implied),
+                    (0x1A, Sign::Implied)
+                ])),
+                Token::OpCode(OpCode(Mnemonic::Nop, AddressingMode::Implied)),
+            ]
+        ))
+    );
+}
+
+#[test]
+fn parse_direct_bytes_db_alias() {
+    let asm = ".BYTE $4E, $45, $53, $1A";
+    let parsed_bytes = super::parse_lines(&asm);
+    let parsed_db = super::parse_lines(&asm);
```

Comment:
I'll assume this was the intension 
```suggestion
    let asm_byte = ".BYTE $4E, $45, $53, $1A";
    let asm_db = ".DB $4E, $45, $53, $1A";
    let parsed_bytes = super::parse_lines(&asm_byte);
    let parsed_db = super::parse_lines(&asm_db);
```